src/pages/home/report/ReportActionCompose.js

```diff
@@ -222,15 +239,34 @@ class ReportActionCompose extends React.Component {
     }
 
     /**
-     * Listens for the keyboard shortcut and submits
-     * the form when we have enter
+     * Listens for keyboard shortcuts and applies the action
      *
      * @param {Object} e
      */
-    triggerSubmitShortcut(e) {
-        if (e && e.key === 'Enter' && !e.shiftKey) {
-            e.preventDefault();
-            this.submitForm();
+    triggerShortcut(e) {
```

Comment:
`processHotkeyActions` does sound better especially considering I used the word "action" in one of the comments.

Might be better with `triggerHotkeyActions` since function itself doesn't process hotkey actions, it instead processes all key presses and triggers the actions.

Reply:
`triggerHotkeyActions` sounds good 👍